lib/jira/commands/install.rb

```diff
@@ -6,6 +6,16 @@ def install
       Command::Install.new.run
     end
 
+    no_commands do
+      def try_install_cookie
+        return if Jira::Core.cookie.empty?
+        puts "  ... detected cookie authentication, renewing your cookie"
```

Comment:
I'm not a big fan of this logic, but it's fine for now I guess.
I think that once we implement a full username / password exchange for a persistent token, we should get rid of session auth altogether.